adapter/internal/oasparser/envoyconf/routes_with_clusters.go

```diff
@@ -114,6 +114,26 @@ func CreateRoutesWithClusters(adapterInternalAPI model.AdapterInternalAPI, inter
 	// creation of clusters.
 	processedEndpoints := map[string]model.EndpointCluster{}
 
+	routeP := CreateAPIDefinitionRoute(adapterInternalAPI.GetXWso2Basepath(), vHost)
+	routes = append(routes, routeP)
+	var endpointForAPIDefinitions []model.Endpoint
+	endpoint := &model.Endpoint{
+		Host:    "localhost",
+		Port:    8084,
+		URLType: "https",
+	}
+	endpointForAPIDefinitions = append(endpointForAPIDefinitions, *endpoint)
+	endpointCluster := model.EndpointCluster{
+		Endpoints: endpointForAPIDefinitions,
+	}
+
+	cluster, address, err := processEndpoints("api_definition_cluster", &endpointCluster, timeout, "")
+	if err != nil {
+		logger.LoggerOasparser.ErrorC(logging.GetErrorByCode(2239, apiTitle, apiVersion, apiDefinitionPath, err.Error()))
```

Comment:
I couldn't find respective error for code `2239`

Reply:
https://github.com/wso2/apk/blob/main/adapter/internal/logging/logging_constant.go#L431